pages/templates/pages/index.html

```diff
@@ -9,28 +9,41 @@
 <div class="container-fluid">
   <div class="row row--main home">
     <div class="container">
-      <section class="module-container clearfix">
+      <section class="module-container module-container--ppl clearfix">
```

Comment:
not sure if `module-container--ppl` does anything useful or if we need it? i copied code from older version of this page that had similar layout.